docs/panel/js/pages/games/games.js

```diff
@@ -110,9 +113,9 @@ $(function () {
                             default:
                                 socket.updateDBValues('adventure_update_settings', {
                                     tables: ['adventureSettings', 'adventureSettings', 'adventureSettings', 'adventureSettings', 'adventureSettings', 'adventureSettings',
-                                        'adventureSettings', 'adventureSettings'],
-                                    keys: ['joinTime', 'coolDown', 'gainPercent', 'minBet', 'maxBet', 'enterMessage', 'warningMessage', 'coolDownAnnounce'],
-                                    values: [joinTime.val(), cooldownTime.val(), gainPercent.val(), minBet.val(), maxBet.val(), entryMessages, userMessages, cooldownMessage]
+                                        'adventureSettings', 'adventureSettings', 'adventureSettings'],
+                                    keys: ['joinTime', 'coolDown', 'gainPercent', 'minBet', 'maxBet', 'enterMessage', 'warningMessage', 'coolDownAnnounce', 'startPermission'],
+                                    values: [joinTime.val(), cooldownTime.val(), gainPercent.val(), minBet.val(), maxBet.val(), entryMessages, userMessages, cooldownMessage, startPermission]
```

Comment:
Never edit the remote panel. It is done via automation

Reply:
Reverted the change